src/Microsoft.OData.Edm/Microsoft.OData.Edm.cs

```diff
@@ -18,14 +18,14 @@ namespace Microsoft.OData.Edm
     /// <summary>
     ///    AutoGenerated resource class. Usage:
     ///
-    ///        string s = EdmRes.GetString(EdmRes.MyIdenfitier);
+    ///        string s = EdmRes.GetString(EdmRes.MyIdentifier);
     /// </summary>
     internal sealed class EdmRes {
         internal const string EdmPrimitive_UnexpectedKind = "EdmPrimitive_UnexpectedKind";
         internal const string EdmPath_UnexpectedKind = "EdmPath_UnexpectedKind";
         internal const string Annotations_TypeMismatch = "Annotations_TypeMismatch";
         internal const string Constructable_VocabularyAnnotationMustHaveTarget = "Constructable_VocabularyAnnotationMustHaveTarget";
-        internal const string Constructable_EntityTypeOrCollectionOfEntityTypeOrComplexTypeExpected = "Constructable_EntityTypeOrCollectionOfEntityTypeOrComplexTypeExpected";
+        internal const string Constructable_EntityTypeOrCollectionOfEntityTypeExpected = "Constructable_EntityTypeOrCollectionOfEntityTypeExpected";

```

Comment:
>Constructable_EntityTypeOrCollectionOfEntityTypeExpected [](start = 30, length = 56)

It's weird, we don't need to change Microsoft.OData.Edm.txt file?

Reply:
I'm unfamiliar with these files. I simply executed the tt script. If that is not enough, let me know!